src/schemas/ProgramDonorSummary/resolvers/summaryEntriesAndStats.ts

```diff
@@ -55,1323 +55,1374 @@ type DonorEntriesResolverType = GraphQLFieldResolver<
+                .gte(1),
+            );
+            break;
+          case workflowStatus.IN_PROGRESS:
+            shouldQueries.push(
+              esb.rangeQuery(EsDonorDocumentField.rnaAlignmentsRunning).gte(1),
+            );
+            break;
+          case workflowStatus.FAILED:
+            shouldQueries.push(
+              esb.rangeQuery(EsDonorDocumentField.rnaAlignmentFailed).gte(1),
+            );
+            break;
+          case workflowStatus.NO_DATA:
+            const mustQueries: Query[] = [];
+            mustQueries.push(
+              esb
+                .rangeQuery(EsDonorDocumentField.rnaAlignmentsCompleted)
+                .lte(0),
+            );
+            mustQueries.push(
+              esb.rangeQuery(EsDonorDocumentField.rnaAlignmentsRunning).lte(0),
+            );
+            mustQueries.push(
+              esb.rangeQuery(EsDonorDocumentField.rnaAlignmentFailed).lte(0),
+            );
+            const noDataQuery = esb.boolQuery().must(mustQueries);
+            shouldQueries.push(noDataQuery);
+            break;
         }
-        const boolQuery = esb.boolQuery().should(sampleQueries);
-        queries.push(boolQuery);
       }
+      const alignmentStatusQuery = esb.boolQuery().should(shouldQueries);
+      queries.push(alignmentStatusQuery);
+    }
 
-      if (
-        field === EsDonorDocumentField.rnaAlignmentStatus &&
-        filter.values.length > 0
-      ) {
-        const shouldQueries: Query[] = [];
-        for (const value of filter.values) {
-          switch (value) {
-            case workflowStatus.COMPLETED:
-              shouldQueries.push(
-                esb
-                  .rangeQuery(EsDonorDocumentField.rnaAlignmentsCompleted)
-                  .gte(1),
-              );
-              break;
-            case workflowStatus.IN_PROGRESS:
-              shouldQueries.push(
-                esb
-                  .rangeQuery(EsDonorDocumentField.rnaAlignmentsRunning)
-                  .gte(1),
-              );
-              break;
-            case workflowStatus.FAILED:
-              shouldQueries.push(
-                esb.rangeQuery(EsDonorDocumentField.rnaAlignmentFailed).gte(1),
-              );
-              break;
-            case workflowStatus.NO_DATA:
-              const mustQueries: Query[] = [];
-              mustQueries.push(
-                esb
-                  .rangeQuery(EsDonorDocumentField.rnaAlignmentsCompleted)
-                  .lte(0),
-              );
-              mustQueries.push(
-                esb
-                  .rangeQuery(EsDonorDocumentField.rnaAlignmentsRunning)
-                  .lte(0),
-              );
-              mustQueries.push(
-                esb.rangeQuery(EsDonorDocumentField.rnaAlignmentFailed).lte(0),
-              );
-              const noDataQuery = esb.boolQuery().must(mustQueries);
-              shouldQueries.push(noDataQuery);
-              break;
-          }
+    if (
```

Comment:
Lines 516 - 531 are the actual changes
Remaining changes are VS Code auto formatting